Client.xcodeproj/xcshareddata/xcschemes/Fennec_Enterprise_SmokeXCUITests.xcscheme

```diff
@@ -361,9 +361,6 @@
                <Test
                   Identifier = "FxScreenGraphTests">
                </Test>
-               <Test
-                  Identifier = "HistoryTests">
```

Comment:
Since the whole History test suite has been removed from here, I can see all these tests enabled in the Smoketest schema. Please disable all except for the created tests. Thanks!